src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  padding: $card-padding;
+
+  &__img {
+    padding: 0 3px;
+    &-img {
+      display: block;
+      background-image: url('../images/imac.jpeg');
+      background-size: cover;
+      height: 134px;
+    }
+  }
+
+  &__header {
+    font-size: $font-size-base;
+    line-height: 18px;
+    font-weight: 500;
+    color: $main-text-color;
```

Comment:
Ensure that the `$main-text-color` variable is defined in a separate SCSS variables file and imported here, as per the task requirements.